User/InMemoryUserRepository.php

```diff
@@ -54,6 +54,18 @@ public function findByUsername($username)
     /**
      * {@inheritDoc}
      */
+    public function find($id)
+    {
+        foreach ($this->users as $user) {
+            if ($user->getId() === $id) {
+                return $user;
+            }
+        }
```

Comment:
what happens in doctrine when it can't find a user ? does it return null ? if that is not the case the result should be mimicked here